includes/REST_API/Stories_Taxonomies_Controller.php

```diff
@@ -159,6 +161,11 @@ public function get_collection_params(): array {
 			'type'        => 'boolean',
 		];
 
+		$query_params['show_ui'] = [
+			'description' => __( 'Whether to show only show in ui taxonomies.', 'web-stories' ),
```

Comment:
```suggestion
			'description' => __( 'Whether to show only show taxonomies that allow a UI in the admin.', 'web-stories' ),
```

Based on the docblocks in `WP_Taxonomy`, improving readability a bit.